Tasks/Common/ios-signing-common/ios-signing-common.ts

```diff
@@ -473,7 +473,7 @@ export async function unlockKeychain(keychainPath: string, keychainPwd: string):
  * @param uuid
  */
 export async function deleteProvisioningProfile(uuid: string): Promise<void> {
-    if (uuid) {
+    if (uuid && uuid.trim()) {
```

Comment:
This could be fixed everywhere with stronger typing.  Something like:

```ts
/** A nonempty string without whitespace */
class Token {
    constructor(public value: string){
        if (value === '' || value.match(/\w/)) {
            throw new Error();
        }
    }
}
```

and then use this as the type for `uuid`, `provProfileUUID`, and `provProfileName`.  I am not sure in this context if we actually want to produce an error or not -- you can use `Token | null` for non-throwing code.